AEPEdgeMedia.xcodeproj/project.pbxproj

```diff
@@ -1380,7 +1380,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.0;
+				MARKETING_VERSION = "";
```

Comment:
why not using the same version here 1.0.0-beta?

Reply:
I noticed that for some of these test targets we use IPHONEOS_DEPLOYMENT_TARGET = 15.5 and TVOS_DEPLOYMENT_TARGET= 15.0. Can we set that to min supported by the sdk 10.0?

Reply:
And for the `MARKETING_VERSION` can we remove that field altogether? The other projects have it only for the lib target, and we may use automation to update these fields which may incorrectly update these

Reply:
I did not find ui option. I can just remove from the project file.